examples/query_example_test.go

```diff
@@ -83,9 +84,9 @@ func runQueryChart() error {
 
 	fmt.Println("### QUERY Chart")
 	agg1 := models.NewAggregate("avg_bits_per_sec", "f_sum_both_bytes", models.AggregateFunctionTypeAverage)
```

Comment:
I don't get why we used `NewAggregate()` here and in the demo we used `Aggregate{}`. Should I make it more consistent?

Reply:
Sorry I meant `kentikapi/internal/resources/query_test.go`, not demo.

Reply:
I suggest to use the constructor in tests as well.